<?xml version="1.0"?>

<launch>
  <!-- face recognition node (recognizes faces in color image and publishes their positions) -->
  <rosparam command="load" ns="/cob_people_detection/face_recognizer" file="$(find cob_people_detection)/ros/launch/face_recognizer_params.yaml"/>
  <node name="face_recognizer" pkg="cob_people_detection" ns="/cob_people_detection/face_recognizer" type="face_recognizer_node" output="screen"> <!--launch-prefix="valgrind"-->
    <remap from="face_positions" to="/cob_people_detection/face_detector/face_positions"/>
	
    <param name="data_directory" type="string" value="$(find cob_people_detection)/common/files/"/>
  </node>

</launch>
